core-build/gulp-core-build/src/tasks/JestTask.ts

```diff
@@ -80,19 +81,31 @@ export class JestTask extends GulpTask<IJestConfig> {
       'config', DEFAULT_JEST_CONFIG_FILE_NAME);
 
     this._copySnapshots(this.buildConfig.srcFolder, this.buildConfig.libFolder);
+
+    let jestConfigFromFile: any = {}; // tslint:disable-line:no-any
+    if (fsx.existsSync(configFileFullPath)) {
+      jestConfigFromFile = JsonFile.load(configFileFullPath);
```

Comment:
>load [](start = 36, length = 4)

Do we want to validate the file against a schema?

Reply:
Jest does schema validation against it. If it is invalid, jest will say something about it. We do not own this config file and therefore do not need to validate it.

---
In reply to: [150124280](https://github.com/Microsoft/web-build-tools/pull/435#discussion_r150124280) [](ancestors = 150124280)